src/main/java/seedu/address/logic/commands/AddStaffCommand.java

```diff
@@ -22,10 +22,11 @@
 import seedu.address.model.staff.Staff;
 
 /**
- * Adds a staff to the address book.
+ * Adds a staff to HR Pro Max++.
```

Comment:
I think it is a good idea to update Javadocs that refer to the address book to our project name.